spec/models/team_spec.rb

```diff
@@ -147,6 +147,86 @@
     end
   end
 
+  describe 'limit of coaches' do
+    let(:team) { create :team }
+    let(:new_user) { create :user }
+    let(:new_coach) { -> {{ name: 'coach', team_id: team.id, user_id: new_user.id }} }
+    let(:new_coach_as_student) { -> {{ name: 'student', team_id: team.id, user_id: new_user.id }} }
+    let(:second_new_coach) { -> {{ name: 'coach', team_id: team.id, user_id: create(:user).id }} }
```

Comment:
Can't all these lambdas (expect the remove coach bit) be transformed to a single `let(:coach_attributes)` that returns a Hash?